tests/Request/ParticipantRequest.php

```diff
@@ -8,18 +8,25 @@
 
 class ParticipantRetrieveRequestTest extends TestCase
 {
+    protected $program = 'sharecare';
```

Comment:
and here 😹

Reply:
I'm going to stop noting it on the other ones, but there are a few more in this PR; i'm sure you get the idea.